### PR TITLE
REP-33: transition to Review

### PR DESCRIPTION
**Pull request by brucexc**
1. Add state diagram based on [REP-33 discussion](https://forum.rss3.io/t/rep-33-node-state-transition/173/14?u=bruce)
2. Completed all states except those involving [REP-38 discussion](https://forum.rss3.io/t/rep-38-demotion-and-slashing-mechanism/175/8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,16 @@ REP stands for RSS3 Evolution Proposal, a detailed documentation proposed by the
...
-| [REP-40](./REPs/REP-40.md) | Whitepaper Updates                             | [pseudoyu](mailto:pseudoyu@connect.hku.hk)                                                                                                                                                                                                                                                              | Core     | Review    |
-| [REP-43](./REPs/REP-43.md) | Introducing Payment Processor for Request Fees | [Nya Candy](mailto:dev@candinya.com)                                                                                                                                                                                                                                                                    | Core     | Review    |
```
Comment (HenryQW) [inline README.md:19]: why is REP-43 removed?

File: REPs/REP-33.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ```
 REP: REP-33
 Title: Node State Transition
-Status: Draft
```
Comment (HenryQW) [inline REPs/REP-33.md:4]: You can't jump to final, see https://github.com/RSS3-Network/REPs/blob/main/REPs/REP-1.md#rep-workflow

**Review by HenryQW — CHANGES_REQUESTED** [review]

**Review by HenryQW — APPROVED** [review]

File: REPs/REP-33.md [hunk 3]
```diff
@@ -17,48 +17,53 @@ Discussions: https://forum.rss3.io/t/proposal-on-node-state-transition/173
...
+7. **Exiting** → (waiting period) → **Exited**: An `Exiting` Node transitions to `Exited` state after completing the waiting period.
+8. **Online**, **Exiting** → **Slashing**: An `Online` or `Exiting` Node transitions to `Slashing` state if its demotion count reaches the threshold. The operator can appeal within 3 epochs.
+9. **Online**, **Exiting** → (current Epoch) → **Offline**: An `Online` or `Exiting` Node transitions to `Offline` state immediately within the same epoch if the Node is detected to be unavailable.
+10. **Offline**, **Slashed** → (current Epoch) → **Initializing**: An `Offline` or `Slashed` Node transitions to `Initializing` state if the Operator manually brings it back online.
+11. **Slashing** → (after 3 epochs) → **Slashed**: A `Slashing` Node transitions to `Slashed` state after 3 epochs.
+12. **Exited** → (anytime) → **Registered**: An `Exited` Node can re-register at any time.
+
+### Reward Mechanism
+
+Nodes in `Online`, `Initializing`, or `Exiting` states are eligible to provide services and receive rewards. Nodes in other states are ineligible.
```
Comment (HenryQW) [inline REPs/REP-33.md]: missing `slashing` state?

**Review by HenryQW — CHANGES_REQUESTED** [review]

File: REPs/REP-33.md (hunk 3 above)
Comment (brucexc) [reply REPs/REP-33.md]: yes, `slashing` state should be rewarded.

**Review by HenryQW — DISMISSED** [review]

**Review by HenryQW — APPROVED** [review]

**Pull request merged**